testsuite/gnat2goto/tests/incomplete_dec/test.adb

```diff
@@ -0,0 +1,8 @@
+with Incomplete_Dec; use Incomplete_Dec;
+
+procedure Test is
+   My_Rec : Partial_Dec := (A=>1);
+begin
+   P(My_Rec);
```

Comment:
What's `P`? `Inc`?

Reply:
P is just an increment routine.  Perhaps better to rename.  In retrospect I need not have made the test for an incomplete type declaration in a package I could have easily written:

procedure New_Incomlete_Dec is
   type Partial_Dec;

   procedure Inc (X : in out Partial_Dec);

   type Partial_Dec is record
      A : Integer;
   end record;

   procedure Inc (X : in out Partial_Dec) is
   begin
      X.A := X.A + 1;
   end Inc;

   V : Partial_Dec := (A => 19);

begin
   Inc (V);
   pragma Assert (V.A = 20);
end New_Incomlete_Dec;

Reply:
Renamed.